de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/48-de.metas.order/5552410_sys_gh6201_add_packing_item_validation_rule_for_invoiceLine.sql

```diff
@@ -0,0 +1,54 @@
+-- 2020-02-17T13:14:26.869Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Val_Rule (AD_Client_ID,AD_Org_ID,AD_Val_Rule_ID,Code,Created,CreatedBy,EntityType,IsActive,Name,Type,Updated,UpdatedBy) VALUES (0,0,540480,'-- GENERAL
+M_HU_PI_Item_Product.AD_Org_ID IN (0, @AD_Org_ID@)
```

Comment:
replace `@M_Product_ID@` with `@M_Product_ID/0@` to avoid errors in the log if this val-rule is run before a product was selected.
Make sure with @dragospodariu96 you both know wtf the `/0` is about